app/models/service_interaction.rb

```diff
@@ -0,0 +1,2 @@
+class ServiceInteraction < ActiveRecord::Base
```

Comment:
Does anyone have any suggestions for a better name for this model? Links will be related to it so it's probably something we'll be referring to frequently.

`Transaction` might work, but I think it's likely to be confusing when we start matching these up to published local transactions (and not all interactions are transactional anyway). Also actual database transactions...

We can rename it later so don't need to find the perfect name right away, but it's worth thinking about now.